### PR TITLE
WFDR-103.1-Landing_Page_User

### PR DESCRIPTION
**Pull request by plee99**
## Summary of changes
created user landing page (without itinerary display implementation)
## Trello Task link
https://trello.com/c/knVeslOw/3-1031-landing-page-user
## Corresponding User Story Number and Title
Use Case#
## Detailed list of changes
page found in static/userLanding.html

Needs to be tested/added with controller:
"plan" button redirecting to view itinerary display

### CODE REVIEW

**Review by setaDM — APPROVED** [review]
Looks good. I'll make sure to add the redirects to the proper endpoints. Just make sure to remerge with devmain.

File: src/main/resources/static/userLanding.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mdb-ui-kit/3.9.0/mdb.min.js" defer></script>
+    <link href="https://cdn.jsdelivr.net/npm/mdb-ui-kit@3.9.0/dist/css/mdb.min.css" rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script src="https://code.jquery.com/ui/1.13.0/jquery-ui.min.js"></script>
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.13.0/themes/base/jquery-ui.css">
+    <title>Wayfinder Admin</title>
```
Comment (NickMoo5) [inline src/main/resources/static/userLanding.html]: This should be updated as it's not admin page

**Review by NickMoo5 — CHANGES_REQUESTED** [review]
Update as per comment please

File: src/main/resources/static/userLanding.html (hunk 1 above)
Comment (plee99) [reply src/main/resources/static/userLanding.html]: updated

**Review by NickMoo5 — APPROVED** [review]
Looks good

**Pull request merged**